### ISSUE TITLE
pgadmin4 Release does not have a Release file. for ubuntu 24.10

### ISSUE THREAD

**Issue by aityahiaidir** [pgadmin-org/pgadmin4#8100]
Hello

i am trying to install Pgadmin on unbuntu 24.10 following the official instruction 

https://www.pgadmin.org/download/pgadmin-4-apt/

Normally there is a release Ubuntu 24.10 (Oracular Oriole), but when a launch this command i get an error below 
```
# Create the repository configuration file:
sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/packages-pgadmin-org.gpg] https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/$(lsb_release -cs) pgadmin4 main" > /etc/apt/sources.list.d/pgadmin4.list && apt update'

```
`error: The repository https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/oracular pgadmin4 Release does not have a Release file.
`
any idea ?
`

**Comment by pravesh-sharma** [comment]
Hi @aityahiaidir,

pgAdmin 4 will be supported on Ubuntu 24.10 from version 8.13 onwards which is due to release on November.

**Comment by aityahiaidir** [comment]
thank's a close the issue

**Comment by aityahiaidir** [comment]
by the way, the version 8.12.1  present in the Ubuntu software center  works perfectly under Ubuntu 24.1

**Comment by adityatoshniwal** [comment]
> by the way, the version 8.12.1 present in the Ubuntu software center works perfectly under Ubuntu 24.1

FYI, pgAdmin is not officially released on Ubuntu software center.

**Comment by aityahiaidir** [comment]
> > by the way, the version 8.12.1 present in the Ubuntu software center works perfectly under Ubuntu 24.1
> 
> FYI, pgAdmin is not officially released on Ubuntu software center.

in any case it is available in the logitech of ubuntu 24.10
![Capture d’écran du 2024-11-14 19-23-49](https://github.com/user-attachments/assets/f5c8a7c1-5346-4bc7-897e-de55b07ac183)
